Sources/RxCocoa/Signal/Signal+Utility.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Signal+Utility.swift
+//  Catalog
+//
+//  Created by Filip Gulan on 13/09/2019.
+//  Copyright © 2019 Infinum. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import RxCocoa
+
+public extension SharedSequenceConvertibleType where SharingStrategy == SignalSharingStrategy {
+
+    /// Creates new subscription and sends elements to `PublishRelay`.
+    ///
+    /// - Parameter relay: PublishRelay instance
+    /// - Returns: Disposable for current subscription
+    func emit(_ relay: PublishRelay<Element>) -> Disposable {
+        return emit(onNext: { e in relay.accept(e) })
+    }
+
+    /// Safely unwraps optional value from chain
+    func unwrap<T>() -> Signal<T> where Element == Optional<T> {
```

Comment:
maybe add a comment that this can be replaced with `compactMap` in versions of RxSwift 5.0.1+

Reply:
Unfortunately still can't. SharedSequences doesn't have `compactMap`.